src/Components/main-view/main-view.jsx

```diff
@@ -1,103 +1,152 @@
-// Import React and useState hook
 import React, { useState, useEffect } from "react";
 import { MovieCard } from "./Components/movie-card/movie-card";
 import { MovieView } from "../movie-view/movie-view";
 import { LoginView } from "../login-view/login-view";
 import { SignupView } from "../signup-view/signup-view";
+import { NavigationBar } from "../navigation-bar/navigation-bar.jsx";
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Button from "react-bootstrap/Button";
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+import { ProfileView } from "../profile-view/profile-view.jsx";
+import "./main-view.scss";
```

Comment:
You added this line, without adding the file. Remove it if you don't yet have the file